src/components/Tally.js

```diff
@@ -116,6 +106,43 @@ const Tally = ({
 
       {(verticalCompact && noticeCount > 0 && showNotices) && <div className={styles.separator} />}
       {(noticeCount > 0 && showNotices) && <Count type='notices' count={noticeCount} horizontal={horizontal} showLabels={showLabels} small={small} verticalCompact={verticalCompact} />}
+    </>
+  )
+
+  const TallyContent = () => (
+    <>
+      {(!horizontal && showLogo && !verticalCompact) && (
+        <img
+          className={classNames(styles.logo, {
+            [styles.logoSmall]: small
+          })}
+          src='https://cdn.scite.ai/assets/images/logo.svg'
+        />
+      )}
+
+      <Counters />
+    </>
+  )
+
+  return (
+    <div
+      className={classNames(classes.tally, {
+        [styles.animatedBorderContainer]: animatedBorder,
+        [styles.noDefaultBorder]: animatedBorder
+      })}
+      onClick={handleClick}
+    >
+      {!animatedBorder && <TallyContent />}
+
+      {animatedBorder && (
+        <>
+          <div className={styles.animatedBorder} />
```

Comment:
I hate to block this but our preference is for custom styling to be done outside of the badge through tagging the CSS.



If that can't be done due to non existing css targets, I would prefer a PR that adds a target for the border instead.

Reply:
Is there a way to solve:
>  I tried in different ways to apply the animation in the element border, but it was impossible by CSS limitations. To make the requested effect, I added a new element that is displayed behind the badge and hiding it overflow, simulating the badge border.

Without having the animation CSS in here?

Reply:
Like we can keep lines 137 to 145 and tag this with a CSS tag no?

Reply:
Here is what I mean:

https://github.com/scitedotai/scite-extension/pull/96

I think that is possible no?

I might be missing something feel free to DM me anytime